[DEBUG] {24/03/2017 17:04:17.977} [http-apr-8080-exec-18] o.s.s.servlet.SigmahHostController - Get the Sigmah welcome page.
[TRACE] {24/03/2017 17:04:19.666} [http-apr-8080-exec-20] o.s.s.d.SecureDispatchServlet - Executing dispatch command.
[DEBUG] {24/03/2017 17:04:19.667} [http-apr-8080-exec-20] o.s.s.s.i.AuthenticationSecureSessionValidator - Starting validation of authentication token 'null' for resource 'org.sigmah.shared.command.SecureNavigationCommand'.
[TRACE] {24/03/2017 17:04:19.667} [http-apr-8080-exec-20] o.s.s.s.i.AuthenticationSecureSessionValidator - No authentication token (anonymous user): 'null'.
[TRACE] {24/03/2017 17:04:19.667} [http-apr-8080-exec-20] o.s.s.s.i.AuthenticationSecureSessionValidator - ACCESS GRANTED for authentication token 'null'.
[TRACE] {24/03/2017 17:04:19.667} [http-apr-8080-exec-20] o.s.s.d.SecureDispatchServlet - COMMAND EXECUTION GRANTED - Command execution: 'org.sigmah.client.security.SecureDispatchAsync$CommandExecution[authToken=null, command=org.sigmah.shared.command.SecureNavigationCommand[page=login], currentPageToken=null]' ; User: 'unknown'.
[DEBUG] {24/03/2017 17:04:19.667} [http-apr-8080-exec-20] o.s.s.dispatch.impl.UserDispatch - EXECUTING COMMAND - Command: 'org.sigmah.shared.command.SecureNavigationCommand[page=login]' ; Handler: 'org.sigmah.server.handler.SecureNavigationCommandHandler@3b4f13d[]' ; User: 'anonymous'.
[TRACE] {24/03/2017 17:04:19.667} [http-apr-8080-exec-20] o.s.s.h.SecureNavigationCommandHandler - ACCESS GRANTED to page 'login' by user 'anonymous'.
[INFO ] {24/03/2017 17:04:19.779} [http-apr-8080-exec-12] o.s.server.servlet.ManifestServlet - Requesting HTML5 manifest for fr@gecko1_8
[DEBUG] {24/03/2017 17:04:30.647} [http-apr-8080-exec-19] o.s.s.servlet.SigmahHostController - Get the Sigmah welcome page.
[INFO ] {24/03/2017 17:04:30.736} [http-apr-8080-exec-11] o.s.server.servlet.ManifestServlet - Requesting HTML5 manifest for fr@gecko1_8
[WARN ] {24/03/2017 17:05:20.788} [C3P0PooledConnectionPoolManager[identityToken->1b60xwc9n9xuzcrbf8hc|499ed621]-HelperThread-#0] c.m.v.r.BasicResourcePool - com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@1ed2e538 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
org.postgresql.util.PSQLException: Connexion refusée. Vérifiez que le nom de machine et le port sont corrects et que postmaster accepte les connexions TCP/IP.
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:138) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:66) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:125) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:30) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.jdbc3g.AbstractJdbc3gConnection.<init>(AbstractJdbc3gConnection.java:22) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.jdbc4.AbstractJdbc4Connection.<init>(AbstractJdbc4Connection.java:32) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.jdbc4.Jdbc4Connection.<init>(Jdbc4Connection.java:24) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.Driver.makeConnection(Driver.java:393) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.Driver.connect(Driver.java:267) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:146) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:195) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:184) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:200) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1086) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1073) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.resourcepool.BasicResourcePool.access$800(BasicResourcePool.java:44) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1810) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:648) [mchange-commons-java-0.2.3.4.jar:na]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_40]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_40]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_40]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_40]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_40]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_40]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_40]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_40]
	at org.postgresql.core.PGStream.<init>(PGStream.java:62) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:76) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	... 17 common frames omitted
[WARN ] {24/03/2017 17:05:20.808} [C3P0PooledConnectionPoolManager[identityToken->1b60xwc9n9xuzcrbf8hc|499ed621]-HelperThread-#2] c.m.v.r.BasicResourcePool - com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@21537346 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
org.postgresql.util.PSQLException: Connexion refusée. Vérifiez que le nom de machine et le port sont corrects et que postmaster accepte les connexions TCP/IP.
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:138) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:66) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:125) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:30) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.jdbc3g.AbstractJdbc3gConnection.<init>(AbstractJdbc3gConnection.java:22) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.jdbc4.AbstractJdbc4Connection.<init>(AbstractJdbc4Connection.java:32) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.jdbc4.Jdbc4Connection.<init>(Jdbc4Connection.java:24) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.Driver.makeConnection(Driver.java:393) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.Driver.connect(Driver.java:267) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:146) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:195) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:184) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:200) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1086) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1073) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.resourcepool.BasicResourcePool.access$800(BasicResourcePool.java:44) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1810) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:648) [mchange-commons-java-0.2.3.4.jar:na]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_40]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_40]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_40]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_40]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_40]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_40]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_40]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_40]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_40]
	at org.postgresql.core.PGStream.<init>(PGStream.java:62) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:76) ~[postgresql-9.1-901-1.jdbc4.jar:na]
	... 17 common frames omitted
[WARN ] {24/03/2017 17:05:21.022} [C3P0PooledConnectionPoolManager[identityToken->1b60xwc9n9xuzcrbf8hc|499ed621]-HelperThread-#0] c.m.v.r.BasicResourcePool - Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@583c3f9 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[WARN ] {24/03/2017 17:05:21.110} [http-apr-8080-exec-20] o.h.e.jdbc.spi.SqlExceptionHelper - SQL Error: 0, SQLState: null
[ERROR] {24/03/2017 17:05:21.110} [http-apr-8080-exec-20] o.h.e.jdbc.spi.SqlExceptionHelper - Connections could not be acquired from the underlying database!
[ERROR] {24/03/2017 17:05:21.112} [http-apr-8080-exec-20] o.s.s.d.SecureDispatchServlet - COMMAND EXECUTION FAILED - Command execution: 'org.sigmah.client.security.SecureDispatchAsync$CommandExecution[authToken=null, command=org.sigmah.shared.command.SecureNavigationCommand[page=login], currentPageToken=null]' ; User: 'unknown' ; RuntimeException while executing.
javax.persistence.PersistenceException: org.hibernate.exception.GenericJDBCException: Could not open connection
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1387) ~[hibernate-entitymanager-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1310) ~[hibernate-entitymanager-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.ejb.QueryImpl.getResultList(QueryImpl.java:277) ~[hibernate-entitymanager-4.2.14.Final.jar:4.2.14.Final]
	at org.sigmah.server.dao.impl.OrgUnitHibernateDAO.getOrgUnitTreeIdsByUserId(OrgUnitHibernateDAO.java:52) ~[OrgUnitHibernateDAO.class:na]
	at org.sigmah.server.handler.SecureNavigationCommandHandler.execute(SecureNavigationCommandHandler.java:95) ~[SecureNavigationCommandHandler.class:na]
	at org.sigmah.server.handler.SecureNavigationCommandHandler.execute(SecureNavigationCommandHandler.java:51) ~[SecureNavigationCommandHandler.class:na]
	at org.sigmah.server.handler.base.AbstractCommandHandler.execute(AbstractCommandHandler.java:117) ~[AbstractCommandHandler.class:na]
	at org.sigmah.server.dispatch.impl.UserDispatch.doExecute(UserDispatch.java:341) ~[UserDispatch.class:na]
	at org.sigmah.server.dispatch.impl.UserDispatch.execute(UserDispatch.java:251) ~[UserDispatch.class:na]
	at org.sigmah.server.dispatch.SecureDispatchServlet.execute(SecureDispatchServlet.java:170) ~[SecureDispatchServlet.class:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_40]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_40]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_40]
	at java.lang.reflect.Method.invoke(Method.java:497) ~[na:1.8.0_40]
	at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:569) [gwt-servlet-2.4.0.jar:na]
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:208) [gwt-servlet-2.4.0.jar:na]
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:248) [gwt-servlet-2.4.0.jar:na]
	at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62) [gwt-servlet-2.4.0.jar:na]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648) [servlet-api.jar:na]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729) [servlet-api.jar:na]
	at com.google.inject.servlet.ServletDefinition.doService(ServletDefinition.java:263) [guice-servlet-3.0.jar:na]
	at com.google.inject.servlet.ServletDefinition.service(ServletDefinition.java:178) [guice-servlet-3.0.jar:na]
	at com.google.inject.servlet.ManagedServletPipeline.service(ManagedServletPipeline.java:91) [guice-servlet-3.0.jar:na]
	at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:62) [guice-servlet-3.0.jar:na]
	at org.sigmah.server.servlet.filter.CacheFilter.doFilter(CacheFilter.java:143) [CacheFilter.class:na]
	at com.google.inject.servlet.FilterDefinition.doFilter(FilterDefinition.java:163) [guice-servlet-3.0.jar:na]
	at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:58) [guice-servlet-3.0.jar:na]
	at com.google.inject.persist.PersistFilter.doFilter(PersistFilter.java:89) [guice-persist-3.0.jar:na]
	at com.google.inject.servlet.FilterDefinition.doFilter(FilterDefinition.java:163) [guice-servlet-3.0.jar:na]
	at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:58) [guice-servlet-3.0.jar:na]
	at com.google.inject.servlet.ManagedFilterPipeline.dispatch(ManagedFilterPipeline.java:118) [guice-servlet-3.0.jar:na]
	at com.google.inject.servlet.GuiceFilter.doFilter(GuiceFilter.java:113) [guice-servlet-3.0.jar:na]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239) [catalina.jar:8.0.22]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206) [catalina.jar:8.0.22]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:219) [catalina.jar:8.0.22]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106) [catalina.jar:8.0.22]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502) [catalina.jar:8.0.22]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:142) [catalina.jar:8.0.22]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79) [catalina.jar:8.0.22]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:617) [catalina.jar:8.0.22]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88) [catalina.jar:8.0.22]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:518) [catalina.jar:8.0.22]
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1091) [tomcat-coyote.jar:8.0.22]
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:668) [tomcat-coyote.jar:8.0.22]
	at org.apache.tomcat.util.net.AprEndpoint$SocketProcessor.doRun(AprEndpoint.java:2463) [tomcat-coyote.jar:8.0.22]
	at org.apache.tomcat.util.net.AprEndpoint$SocketProcessor.run(AprEndpoint.java:2452) [tomcat-coyote.jar:8.0.22]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_40]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_40]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.0.22]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_40]
Caused by: org.hibernate.exception.GenericJDBCException: Could not open connection
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:54) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:124) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:109) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.obtainConnection(LogicalConnectionImpl.java:221) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.getConnection(LogicalConnectionImpl.java:157) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.connection(StatementPreparerImpl.java:56) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$5.doPrepare(StatementPreparerImpl.java:159) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:183) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareQueryStatement(StatementPreparerImpl.java:157) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.loader.Loader.prepareQueryStatement(Loader.java:1881) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1858) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1838) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.loader.Loader.doQuery(Loader.java:906) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:348) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.loader.Loader.doList(Loader.java:2550) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.loader.Loader.doList(Loader.java:2536) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2366) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.loader.Loader.list(Loader.java:2361) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.loader.hql.QueryLoader.list(QueryLoader.java:495) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.list(QueryTranslatorImpl.java:357) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.engine.query.spi.HQLQueryPlan.performList(HQLQueryPlan.java:198) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.internal.SessionImpl.list(SessionImpl.java:1194) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.internal.QueryImpl.list(QueryImpl.java:101) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.ejb.QueryImpl.getResultList(QueryImpl.java:268) ~[hibernate-entitymanager-4.2.14.Final.jar:4.2.14.Final]
	... 47 common frames omitted
Caused by: java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118) ~[mchange-commons-java-0.2.3.4.jar:na]
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:689) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at org.hibernate.service.jdbc.connections.internal.C3P0ConnectionProvider.getConnection(C3P0ConnectionProvider.java:84) ~[hibernate-c3p0-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.internal.AbstractSessionImpl$NonContextualJdbcConnectionAccess.obtainConnection(AbstractSessionImpl.java:301) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	at org.hibernate.engine.jdbc.internal.LogicalConnectionImpl.obtainConnection(LogicalConnectionImpl.java:214) ~[hibernate-core-4.2.14.Final.jar:4.2.14.Final]
	... 67 common frames omitted
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1418) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:606) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:526) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:755) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:682) ~[c3p0-0.9.2.1.jar:0.9.2.1]
	... 71 common frames omitted
[WARN ] {24/03/2017 17:05:21.113} [C3P0PooledConnectionPoolManager[identityToken->1b60xwc9n9xuzcrbf8hc|499ed621]-HelperThread-#2] c.m.v.r.BasicResourcePool - Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@583c3f9 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.